shared/charts/gpii-productiontests/values.yaml

```diff
@@ -0,0 +1,23 @@
+# Default values for gpii-productiontests.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+couchdb:
+  url: "http://admin:password@couchdb-svc-couchdb.gpii.svc.cluster.local:5984/gpii"
+  labels:
+    app: couchdb
+  port: http-couchdb
+
+flowmanager:
+    url: http://flowmanager.gpii.svc.cluster.local:80
+    labels:
+        app: flowmanager
+    svcListenport: http
+    flowmanagerListenPort: 8080
```

Comment:
This block ^ seems to have extra indentation.

Reply:
Good eye, @stepanstipl !

I'll fix it.